src/tests/estimation/generate_tests_results.py

```diff
@@ -0,0 +1,58 @@
+
+import numpy as np
+
+from med_bench.get_simulated_data import simulate_data
+from med_bench.get_estimation import get_estimation
+
+from med_bench.utils.constants import ESTIMATORS, PARAMETER_LIST, PARAMETER_NAME
+
+
+def get_data_from_list(data):
```

Comment:
I would suggest to prepend an underscore for 'local' functions, i.e. `_get_data_from_list`
An have a one-line docstring just to explain what this does.